lib/service_actor/checks/default_check.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+# Adds the `default:` option to inputs. Accepts regular values and lambdas.
+# If no default is set and the value has not been given, raises an error.
+#
+# Example:
+#
+#   class MultiplyThing < Actor
+#     input :counter, default: 1
+#     input :multiplier, default: -> { rand(1..10) }
+#   end
+#
+#   class MultiplyThing < Actor
+#     input :counter,
+#           default: {
+#             is: 1,
+#             message: "Counter is required"
+#           }
+#
+#     input :multiplier,
+#           default: {
+#             is: -> { rand(1..10) },
+#             message: (lambda do |input_key:, actor:|
+#               "Input \"#{input_key}\" is required"
+#             end)
+#           }
+#   end
+class ServiceActor::Checks::DefaultCheck < ServiceActor::Checks::Base
+  def self.check(result:, input_key:, input_options:, actor:, **)
+    new(
+      result: result,
+      input_key: input_key,
+      input_options: input_options,
+      actor: actor,
+    ).check
+  end
+
+  def initialize(result:, input_key:, input_options:, actor:)
+    super()
+
+    @result = result
+    @input_key = input_key
+    @input_options = input_options
+    @actor = actor
+  end
+
+  def check # rubocop:disable Metrics/MethodLength
```

Comment:
If this _returns_ a single error, perhaps we can rename the method to `#error`?

Reply:
This name has the idea that the method checks something and in which case it either returns `nil` or an error. That is, here I laid down what "needs to be done" (check), and not what "returns" (an error or `nil`).